dom0/sd-workstation.top

```diff
@@ -9,14 +9,19 @@ base:
     - sd-upgrade-templates
     - sd-dom0-qvm-rpc
     - sd-sys-whonix-vms
+    - sd-log
     - sd-devices
     - sd-gpg
     - sd-proxy
     - sd-viewer
     - sd-app
     - sd-whonix
     - sd-remove-unused-templates
-    - sd-log
+
+  sd-log-buster-template:
+    - sd-log-template-files
+  sd-log:
```

Comment:
I don't think this is required since you are shutting down this machine after provisioning the template